web_set_single_page_hidden/static/src/js/relational_fields.js

```diff
@@ -0,0 +1,29 @@
+odoo.define("web_set_single_page_hidden.relational_fields", function(require) {
+    "use strict";
+
+    var relational_fields = require("web.relational_fields");
+    var FieldOne2Many = relational_fields.FieldOne2Many,
+        FieldMany2Many = relational_fields.FieldMany2Many;
+
+    FieldOne2Many.include({
+        start: function() {
+            var self = this;
+            return this._super.apply(this, arguments).then(function() {
+                if (!_.isUndefined(self.pager)) {
+                    self.pager.set_single_page_hidden(false);
```

Comment:
It would be nice to have this configurable from the field options.
Something like: `self.pager.set_single_page_hidden(self.options.single_page_hidden);`
Is this possible?

Reply:
it is possible. but don't you think that it should be handled as new PR and not in Migration PR?

Reply:
:+1: